doc/source/user_guide/numpyro_refitting_xr_lik.ipynb

```diff
@@ -2090,11 +2091,11 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "We are now missing the `log_likelihood` group because we have not used the `log_likelihood` argument in `idata_kwargs`. We are doing this to ease the job of the sampling wrapper. Instead of going out of our way to get Stan to calculate the pointwise log likelihood values for each refit and for the excluded observation at every refit, we will compromise and manually write a function to calculate the pointwise log likelihood.\n",
+    "We are now missing the `log_likelihood` group because we have not used the `log_likelihood` argument in `idata_kwargs`. We are doing this to ease the job of the sampling wrapper. Instead of going out of our way to get NumPyro to calculate the pointwise log likelihood values for each refit and for the excluded observation at every refit, we will compromise and manually write a function to calculate the pointwise log likelihood.\n",
```

Comment:
Here it will be "Instead of going out of our way to get Stan to calculate the pointwise log likelihood values for each refit ". The Stan will remain as it is.

Reply:
Done